README.md

```diff
@@ -0,0 +1,67 @@
+# GBoard4A
+
+[![Build and Test](https://github.com/labcif/GBoard4A/actions/workflows/workflow.yml/badge.svg)](https://github.com/labcif/GBoard4A/actions/workflows/workflow.yml)
+[![codecov](https://codecov.io/gh/labcif/GBoard4A/branch/main/graph/badge.svg?token=XD675L6X9K)](https://codecov.io/gh/labcif/GBoard4A)
+
+GBoard4A is forensic analyzer for Autopsy digital forensic software to analyze GBoard application data.
+
+## Requirements
+
+* A D compiler (e.g.: `dmd`)
+* `dub` package manager
+* `autopsy` at least version 4.17
+* `sqlite3` library
```

Comment:
not needed

Reply:
What do you mean? I can simplify this to the D toolchain and `sqlite3`.

Reply:
`sqlite3` does not need to be manually downloaded to use this module as it is only used in the D application.

Reply:
> `sqlite3` does not need to be manually downloaded to use this module as it is only used in the D application.

It needs. D package is just a wrapper to the C library, it still needs the actual system library unless, on Windows, you use the bundled libraries in the repository https://github.com/dlang-community/d2sqlite3/tree/v1.x.x/lib . Otherwise, undefined reference happens on linking.

> `with-lib` (the default): assumes that SQLite is already installed and available to the linker.

https://github.com/dlang-community/d2sqlite3#dub-configurations

Reply:
true